test/common/network/address_impl_test.cc

```diff
@@ -315,8 +315,7 @@ TEST(AddressFromSockAddr, Pipe) {
 
   // Empty path (== start of Abstract socket name) is invalid.
   StringUtil::strlcpy(sun.sun_path, "", sizeof sun.sun_path);
-  EXPECT_THROW(addressFromSockAddr(ss, sizeof(sa_family_t) + 1 + strlen(sun.sun_path)),
-               EnvoyException);
+  EXPECT_THROW(addressFromSockAddr(ss, offsetof(struct sockaddr_un, sun_path) + 1), EnvoyException);
```

Comment:
To make this less magic can it be the same as rest of code which is offsetof(struct sockaddr_un, sun_path) + 1 + strlen(sun.sun_path) still?